install/helm/open-match/templates/backend.yaml

```
@@ -71,5 +71,5 @@ spec:
           containerPort: {{ .Values.openmatch.backend.grpc.port }}
         - name: http
           containerPort: {{ .Values.openmatch.backend.http.port }}
-        {{- include "kubernetes.probe" (dict "port" .Values.openmatch.backend.http.port) | nindent 8 }}
+        {{- include "kubernetes.probe" (dict "port" .Values.openmatch.backend.http.port "https" .Values.openmatch.tls.enabled) | nindent 8 }}
```

Maybe something like
```suggestion
        {{- include "kubernetes.probe" (dict "port" .Values.openmatch.backend.http.port "isHTTPS" .Values.openmatch.tls.enabled) | nindent 8 }}
```